### DISCUSSION TITLE
File name and Files missing in Istanbul coverage report

### DISCUSSION THREAD

**Discussion by SujithGunasekaran** [vitest-dev/vitest#2214] [Q&A]
Hi Team, 

I am using _**@vitest/coverage-istanbul**_ for the unit test coverage report. I am getting null as the file name in the coverage report. and also I am having the file called **_UserCart.vue_** and added a unit test for that file. But the UserCart.vue file alone is not showing in the coverage report.

<img width="942" alt="Screenshot 2022-10-27 at 16 12 04" src="https://user-images.githubusercontent.com/68234378/198275902-559149e6-e4cd-4cfd-bd86-17eb1c0626e4.png">

Below is my test configuration in vitest.confi.js

```
test: {
      environment: 'jsdom',
      coverage: {
        all: true,
        provider: 'istanbul',
        include: ['**/*.{js,vue}'],
        exclude: [
          '**/node_modules/**',
        ],
        reporter: ['clover', 'html', 'text', 'text-summary', 'lcov'],
      },
    }
```

Can someone please help me to resolve the issues?

**Comment by AriPerkkio** [comment]
@SujithGunasekaran This should be now fixed and released in [`0.25.3`](https://github.com/vitest-dev/vitest/releases/tag/v0.25.3). Fixed by #2338.
